Docs update (#30)
nufi-official/nufi-dapp-sdk: docs/common/widgetOptions.md

  // whatever else
}
```

## Featuring custom tokens

> You can "feature" tokens that you dapp manages to be displayed with bigger priority in the

Reviewer: ```suggestion
You can "feature" tokens that you dapp manages to be displayed at the top of the
```

user tokens list section. Note that you can feature one or two tokens. If you try to feature more,

> other tokens will be ignored.

Reviewer: ```suggestion
the rest of the tokens would be ignored.
```

Also note that if user has no funds in the wallet no tokens are listed.

> The option is being controlled using `initNufiDappCardanoSdk`:

Reviewer: ```suggestion
This option can be controlled from the `initNufiDappCardanoSdk()` call:
```

```typescript
initNufiDappCardanoSdk(nufiCoreSdk, platform, {
    featuredTokens: [{
      assetNameHex: 'YOUR_ASSET_NAME_HEX',
      policyIdHex: 'YOUR_POLICY_ID_HEX'
    }]
  })
```

## Color mode

> You can choose from `dark` and `light` color mode. Note that we default to `dark`.

Reviewer: ```suggestion
You can choose from `dark` and `light` color mode. Note that default value is `dark`.
```

The color mode you choose will only take effect if users did not change it on their

> own, then users choice takes precedence.

Reviewer: ```suggestion
own, otherwise user's choice takes precedence.
```

```typescript
nufiCoreSdk.init(nufiDomain, {
  colorMode: 'dark' | 'light'
})
```

## Manual Widget collapse/expand & visibility status

You can use the following methods to get widget visibility status on demand,
and open/close it on your own. This can be handy if your dapp manages more complex
UX flow, otherwise we suggest you to avoid using these methods.

> Note that as of now we do not allow to subscribing to widget visibility status.

Reviewer: ```suggestion
Note that as of now it is not possible to subscribe to the widget visibility status changes.
```
